dotCMS/src/integration-test/java/com/dotcms/rest/api/v1/apps/AppsResourceTest.java

```diff
@@ -141,44 +141,44 @@ public void Test_Create_service_descriptor_Then_Create_Service_Integration_Then_
 
         when(request.getRequestURI()).thenReturn("/baseURL");
 
-        final String serviceKey = String.format("lol_%d", System.currentTimeMillis());
-        final String fileName = String.format("%s.yml", serviceKey);
-        try(final InputStream inputStream = createServiceDescriptorFile(fileName, serviceKey, "lola",
+        final String appKey = String.format("lol_%d", System.currentTimeMillis());
+        final String fileName = String.format("%s.yml", appKey);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "%s.yml" appears 7 times in this file; the first occurrence is on line 145](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5218098)